awx/main/access.py

```diff
@@ -663,6 +663,8 @@ def can_delete(self, obj):
             return False
         if self.user.is_superuser:
             return True
+        if self.can_admin(obj, None):
+            return True
```

Comment:
I might lump a couple of comments together here

 - I would pass `allow_orphans=True` to this. Why? Because an org admin can create a user not in an organization (using the CLI), and this would allow them to undo this while still not giving away too much. The ability to undo what you have done is kind of the theme of the issue.
 - remove the prior `if self.user.is_superuser` check right above this line. `can_admin` already checks that (decorator)
 - we should put in a unit test

Reply:
On it :smile:

Reply:
All done except the unit test.